server/db/index.js

```diff
@@ -1,13 +1,204 @@
 exports.users = require('./users');
 
 exports.getOpenSimulations = function (instructor_token) {
+  /**
+   * http://localhost:3000/api/v1/dashboard
+  */
   return [];
 }
 
 exports.getClosedSimulations = function (instructor_token) {
+  /**
+   * http://localhost:3000/api/v1/dashboard
+  */
   return [];
 }
 
 exports.getDraftedSimulations = function (instructor_token) {
+  /**
+   * http://localhost:3000/api/v1/dashboard
+  */
   return [];
 }
+
+exports.getSimulationSummary = function (instructor_token) {
```

Comment:
I think parameter `instructor_token` is not appropriate on this function, and also parameter `token` on the following functions. Parameters of these functions must consider what type of inputs would be the arguments for the purpose of each function. For example, in `exports.getSimulationSummary`, I think the parameter should look like this:

```suggestion
exports.getSimulationSummary = function (simulation_uid) {
```

Since `instruction`, `initial-reflection`, and `initial-action` are components of a simulation, we can come up with the fact that we need UID for a simulation to get the summary of it. I think you can change the parameters of the other functions. You can look up on schema document or endpoint document in google docs and Postman to check what input or output of any function is.